eva-accession-pipeline/src/main/resources/monitoring/monitor_duplicate_accessions.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 EMBL - European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+from datetime import datetime
+import getpass
+import smtplib
+from __init__ import *
+
+
+def export_mongo_accessions(mongo_connection_properties, collection_name, export_output_filename):
+    export_command = 'mongoexport --host {0} --port {1} --db {2} --username {3} --password {4} ' \
+                     '--authenticationDatabase={5} --collection {6} --type=csv --fields accession -o "{7}" '\
+                    .format(mongo_connection_properties["mongo_host"],
+                            mongo_connection_properties["mongo_port"],
+                            mongo_connection_properties["mongo_db"],
+                            mongo_connection_properties["mongo_username"],
+                            mongo_connection_properties["mongo_password"],
+                            mongo_connection_properties["mongo_auth_db"],
+                            collection_name,
+                            export_output_filename) + " --sort '{accession: 1}'"
```

Comment:
The previous string is delimited by single quotes and the json string argument for sort is also single quotes so it would have to be escaped... i felt that it might look messy that way so I separated them...

Reply:
I pointed it out because I was expecting a sort somewhere and I didn't find it at first. IMO keeping that together is more important than avoiding quote mess.

I would prefer either:
- escaping: 
```
'single \'quotes\'' \
'double "quotes"'
```
- or using triple quotes (note the last space to avoid 4 quotes):
```
"""single \'quotes\'' \
"""double "quotes" """
```
- or changing the outer quotes from each line.
```
"single 'quotes'"
'double "quotes"'
```

Reply:
Resolved...